modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+ifeval::["{context}" == "installing-aws-localzone"]
+:localzone:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="installation-extend-edge-nodes-aws-local-zones_{context}"]
+= Creating user workloads in AWS Local Zones
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create a user workloads in Local Zone subnets. 
+
+After the `openshift-installer` creates the cluster, the installer automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod to a node if the pod's deployment does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+
+The `openshift-installer` creates the Machine Set manifest and attaches `node-role.kubernetes.io/edge` and `node-role.kubernetes.io/worker` labels to each edge worker node that is located in a Local Zone subnet.
+
+
+.Prerequisites
+
+* You have an access to the OpenShift CLI (`oc`).
```

Comment:
@mtulio . I should mention the creation of the worker manifest file here and the importance of defining the taint?

Reply:
> * You configured an AWS Local Zones environment. 

I think we must improve this line with something like a cluster deployed extending works to Local Zones (currently day-0 docs).

With regards the `nd the importance of defining the taint`, not sure about defining the importance as it should be a concept of K8s. The taints is a design choice, the user can deploy a cluster without taints and tolerations, but we decide that architecture to avoid spreading cluster components in more expensive locations, impacting/increasing in costs and latency.

Also if the user followed our documentation, the worker nodes must be tainted, so maybe is worth to mentioned that we expected the nodes to be tainted to match the tolerations mentioned on the use case/deployment.
wdyt?

Reply:
Thanks, @mtulio . Yes, good suggestion. I also removed a bullet point, because it melds with my new bullet point.

I updated the bullet point to state:

>  You deployed your cluster in a Virtual Private Cloud (VPC) with defined Local Zone subnets for extending regular worker nodes to edge worker nodes